package.json

```diff
@@ -3,18 +3,24 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
+    "@emotion/styled": "^10.0.23",
     "@material-ui/core": "^4.4.1",
     "@material-ui/icons": "^4.4.3",
     "@material-ui/lab": "^4.0.0-alpha.27",
+    "@projectstorm/react-diagrams": "^5.3.2",
+    "closest": "0.0.1",
     "husky": "^3.0.5",
     "lint-staged": "^9.2.5",
     "lodash": "^4.17.15",
+    "mathjs": "^6.2.5",
```

Comment:
Are you using these package ?

Reply:
Yep,  `@emotion/styled`, `closest`, `lodash`, `mathjs`, `pathfinding`, and `paths-js` are all peer dependencies of `react-diagrams`